tests/test_commands.py

```diff
@@ -44,7 +45,28 @@
         'iprop': 1,
         'idf': 1,
         'kcmplx': 1
-    }
+}
+
+
+def test_cmd_class():
```

Comment:
Note:
I've moved the command output and command instance to `test_cmd_class`. Limit the scope of variables as much as possible. If there was a single test, it's ok to have a module constant, but it doesn't scale since as soon as you start adding more tests/functions to a module you'll have more `CONSTANTS` flying around. Instead, the better approach is to either use a fixture that returns a `CommandOutput`, or since we only have one test, just instantiate `CommandOutput` within the test itself.

Reply:
Ok. Makes sense. Will try to remember in the future.